src/less/month.less

```diff
@@ -28,6 +28,7 @@
   height: auto;
   line-height: normal;
   white-space: nowrap;
+  pointer-events: all;
```

Comment:
?

Reply:
This point is important.
As `.rbc-row-content` is overlaying `.rbc-day-bg` (in month view), that is listening to `doubleClick` events, we have to add `pointer-events: none` to `.rbc-row-content`, and `pointer-events: all` to all active elements inside it (events, popup link, day number link)

Reply:
I think that may be a problem, first it won't work in older browsers and second I think it is going to break the drag and drop code as well, since they also mess with pointer-events. 

Maybe we can narrow the scope of this PR a bit, and only add doubleClick handlers where there is already a click handler, that will make sure we have a nice parity and we can talk about adding new handlers in a separate PR/issue?

Reply:
No, I checked. It doesn't make problems with DnD. Moreover, `pointer-events` properties are already used in some places of DnD addon. I can't see any other solutions, how to catch `doubleClick` events under `.rbc-row-content` overlay (remember, that clicked date should be calculated and passed to handler. this is a limitation)

Ok, I'll leave `onDoubleClickEvent` in this PR, and `onDoubleClickSlot` and `onDoubleClickAllDaySlot` will be moved to another one.